tests/usdt_sample/scripts/latency.c

```diff
@@ -0,0 +1,127 @@
+#include <linux/blkdev.h>
+#include <uapi/linux/ptrace.h>
+
+// The following methods were taken from the verbose output of argdist.py.
+// Function names were adjusted to match the BPF function name expectations.
+static inline int _bpf_readarg_trace_operation_start_1(struct pt_regs* ctx, void* dest, size_t len)
```

Comment:
Thanks for the feedback, I'll get back to it shortly with the following changes (summary of feedback so far):
- Move usdt_sample to examples (instead of tests).
- Use the generated code for the _bpf_readarg_xxx methods (using the suggested debug=4 option).
- Use folly usdt headers to avoid systemtap-sdt-devel dependency. I'll add some documentation explaining that systemtap-sdt-devel can be used as an alternative to the usdt headers.